sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -230,7 +230,7 @@
       "deprecation": {
         "level": "error",
         "reason": "Use uniform property format(spring.cloud.azure.*) in all Spring Cloud Azure libraries.",
-        "replacement": "spring.cloud.azure.active-directory.base-uri"
+        "replacement": "spring.cloud.azure.active-directory.profile.environment.active-directory-endpoint"
```

Comment:
Are we also going to update the migration guide?

Reply:
Already created a PR: https://github.com/microsoft/spring-cloud-azure/pull/786